dependencies.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+subprocess.check_call([sys.executable, '-m', 'ensurepip'])
+logging.info("Pip is installed")
+
+# Make sure that dependencies are installed
+logging.info("Installing dependencies...")
+subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-r', 'requirements.txt'])
+subprocess.check_call(['git', 'init'])
```

Comment:
What should happen if git is not installed?

Reply:
Resolved by catching exception and stopping dependencies.py